build.py

```diff
@@ -38,9 +38,20 @@ def build_index(full_path, package_name, apps):
     content = python(os.path.join(app['folder'], 'build_index.py'))
     return content
 
+@chmod(0o755)
+@file
+def postinst(full_path, package_name, apps):
+    domains = ','.join(apps[package_name]['https_domains'])
+    return f'''\
+        #!/bin/bash
+        set -x
+        sudo certbot run -n --nginx --agree-tos -d {domains}
```

Comment:
I expect some nginx.conf settings to point to where the SSL lives? Doesn't certbot need to be initiated first?